src/config.ts

```diff
@@ -56,9 +58,7 @@ export const defaultRadius = 2000; //in meters
 export const allowLocalhostUI = Boolean(
   process.env.ALLOW_LOCALHOST_UI === 'true'
 );
-export const allowHardhatNetwork = Boolean(
-  process.env.ALLOW_HARDHAT_NETWORK === 'true'
-);
+export const networkMode = String(process.env.NETWORK_MODE) as NetworkMode;
 
 export const assetsCurrencies = ['EUR', 'USD', 'JPY', 'CNY']; // currency the asset is pegged to
 export type AssetCurrency = typeof assetsCurrencies[number];
```

Comment:
this and some other definitions can be removed because they are already defined in the `win-commons/dist/config`